ExplorerTests/ExplorerTests.swift

```diff
@@ -21,16 +21,39 @@ class ExplorerTests: XCTestCase {
         super.tearDown()
     }
     
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    func testInitDiveLogViewModel() {
+        var dateComponents = DateComponents()
+        dateComponents.year = 2017
+        dateComponents.month = 02
+        dateComponents.day = 28
+        dateComponents.timeZone = TimeZone(abbreviation: "PST")
+        dateComponents.hour = 13
+        dateComponents.minute = 38
+        
+        let userCalendar = Calendar.current
+        let date = userCalendar.date(from: dateComponents)
+        
+        let timeOut = date!.addingTimeInterval(1800)
+        
+        let timeOutDate = userCalendar.date(from: timeOutDateComponents)
```

Comment:
I would be cool to replace lines 25-38 with a helper function that returns two dates, one being a given duration later than the other. This function could return a tuple:

```
func datesSeparated(by duration: Double) -> (Date, Date) {
        var dateComponents = DateComponents()
        dateComponents.year = 2017
        dateComponents.month = 02
        dateComponents.day = 28
        dateComponents.timeZone = TimeZone(abbreviation: "PST")
        dateComponents.hour = 13
        dateComponents.minute = 38
        
        let userCalendar = Calendar.current

        let timeIn = userCalendar.date(from: dateComponents)
        let timeOut = timeIn!.addingTimeInterval(duration)

        return (timeIn, timeOut)
}

let times = datesSeparated(by: 1800)
//Access the dates like this: times.0 & times.1
```
A couple of things to notice:
1. The function names. They should read like natural language. The word "by" is actually what's called a "label." I want to add it to make the method signature make sense, but I don't want to add it to the name of the parameter or the method name. It's a cool lil feature of Swift.

2. I'm returning a tuple as the return value. You can really return any set of values or types in a tuple. they're pretty cool.